cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -231,11 +231,17 @@ bundle agent cfe_internal_permissions
     "_cf_execd_socket_allow_users"
       slist => { @(def.control_executor_runagent_socket_allow_users) };
 
+    "_cf_statedir_allow_users"
+      slist => { "cfpostgres", "cfapache", @(def.control_executor_runagent_socket_allow_users) },
```

Comment:
Do we need to de-duplicate this list or is it okay if it has duplicates? I guess you tested the defaults which actually have a duplicate `cfapache` item in the list?

Reply:
In my testing, it was fine to be duplicated.